types/auth0-js/index.d.ts

```diff
@@ -536,7 +536,23 @@ export interface AuthOptions {
      */
     maxAge?: number;
     leeway?: number;
-    plugins?: any[];
+    jwksURI?: string;
+    overrides?: {
+      __tenant: string;
+      __token_issuer: string;
+    };
+    plugins?: any;
+    popupOrigin?: string;
+    protocol?: string;
+    response_type?: string;
+    rootUrl?: string;
```

Comment:
just wondering if `rootUrl ` can be `write`?:
https://github.com/auth0/auth0.js/blob/c35e6c18f905a0fca0dcb6edca129ea6661898f4/src/authentication/index.js#L92

Reply:
Im not clear what you mean by if this can be `write`? 

I see the `rootUrl` is set based on `domain` so I understand there's no point of setting `rootUrl` at all. Is that what you meant?

Reply:
@peterblazejewicz ping

Reply:
yes, that was my concnern: should it be possible to assign this value. I can now see they are writing this value anyway (e.g. in tests). So up to you, closing this.

Reply:
removed it.